src/pages/installation/cdn.md

```diff
@@ -10,44 +10,121 @@ contributors:
-<link href="https://unpkg.com/@ionic/core@latest/css/ionic.bundle.css" rel="stylesheet">
+/* Basic CSS for apps built with Ionic */
+@import "~@ionic/angular/css/normalize.css";
+@import "~@ionic/angular/css/structure.css";
+@import "~@ionic/angular/css/typography.css";
+
+/* Optional CSS utils that can be commented out */
+@import "~@ionic/angular/css/padding.css";
+@import "~@ionic/angular/css/float-elements.css";
+@import "~@ionic/angular/css/text-alignment.css";
+@import "~@ionic/angular/css/text-transformation.css";
+@import "~@ionic/angular/css/flex-utils.css";
+@import "~@ionic/angular/css/display.css";
 ```
 
-With this it's possible to use all of the Ionic components without having to install anything.
+See [Global Stylesheets](../layout/global-stylesheets) for the styles that each of these files contain.
 
-<blockquote>
-  <p>
-    This does not install Angular or any frameworks. This allows use of Ionic components without a framework.
-  </p>
-</blockquote>
 
+## Stencil + Ionic Framework
 
-## Ionicons from a CDN
+When using Ionic Framework in a Stencil project, install the latest `@ionic/core` package from [npm](/docs/faq/glossary#npm). This comes with all of the Ionic Framework components.
 
-Ionicons is packaged by default with the Ionic Framework, so no installation is necessary if you're using Ionic. Want to use Ionicons without Ionic Framework? Place the following `<script>` near the end of your page, right before the closing `</body>` tag, to enable them.
+```shell
+$ npm install @ionic/core@latest --save
+```
+
+### JS
+
+Include an import of `@ionic/core` in the root app file:
+
+```typescript
+import '@ionic/core';
+```
+
+### CSS
+
+Create a global stylesheet file and add it to the `config` object in the [Stencil Config](https://stenciljs.com/docs/config).
+
+```javascript
+exports.config = {
+  // ...
+
+  globalStyle: 'src/global.css'
+
+  // ...
+};
+```
+
+Add the following imports to that file:
+
+```css
+/* Core CSS required for Ionic components to work properly */
+@import "~@ionic/core/css/core.css";
+
+/* Basic CSS for apps built with Ionic */
+@import "~@ionic/core/css/normalize.css";
+@import "~@ionic/core/css/structure.css";
+@import "~@ionic/core/css/typography.css";
+
+/* Optional CSS utils that can be commented out */
+@import "~@ionic/core/css/padding.css";
+@import "~@ionic/core/css/float-elements.css";
+@import "~@ionic/core/css/text-alignment.css";
+@import "~@ionic/core/css/text-transformation.css";
+@import "~@ionic/core/css/flex-utils.css";
+@import "~@ionic/core/css/display.css";
+```
+
+See [Global Stylesheets](../layout/global-stylesheets) for the styles that each of these files contain.
+
+
+## Ionicons CDN
+
+Ionicons is packaged by default with the Ionic Framework, so no installation is necessary if you're using Ionic. To use Ionicons without Ionic Framework, place the following `<script>` near the end of your page, right before the closing `</body>` tag, to enable them.
```

Comment:
Probably worth linking the [Ionicons usage page](https://ionicons.com/usage) somewhere here for detailed instructions.